tools/autodeployment/webhook.js

```diff
@@ -53,12 +53,12 @@ function requestFilter(repo, buildType, action, ref, mainBranch) {
  * @param {'push' | 'release'} eventType one of 'push' or 'release'
  * @param {Object} githubData webhook payload from GitHub, see https://github.com/Seneca-CDOT/telescope/settings/hooks
  */
-function getSha(eventType, githubData) {
-  if (eventType === 'push') {
+function getSha(deployTag, githubData) {
+  if (deployTag === 'staging') {
```

Comment:
We'll have to see when we test if the payload matches this same style.  I suspect this function will need to change.  NOTE: we should be able to use `.commit` based on https://github.com/distributhor/workflow-webhook, but let's see when we test on staging.